wrappers/python/pyrs_device.cpp

```diff
@@ -52,6 +53,12 @@ void init_device(py::module &m) {
                 ss << "  on USB" << self.get_info( RS2_CAMERA_INFO_USB_TYPE_DESCRIPTOR );
             ss << ")>";
             return ss.str();
+        })
+        .def("is_metadata_enabled", [](const rs2::device& self) -> bool{
+            auto depth_sens = self.first< rs2::depth_sensor >();
+            std::string id = depth_sens.get_info(RS2_CAMERA_INFO_PHYSICAL_PORT);
```

Comment:
Might want to check if the info is supported, or it will throw, right?

Reply:
What should I do if it isn't? return false?

Reply:
Throw an exception